src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts

```diff
@@ -2382,8 +2382,6 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditorD
 				if (typeof options?.focusEditorLine === 'number') {
 					this._cursorNavMode.set(true);
 					this.revealInView(cell);
-				} else if (options?.revealBehavior === ScrollToRevealBehavior.fullCell) {
```

Comment:
Just double checking, I'm assuming this behavior is covered by `revealInCenterIfOutsideViewport(cell)` and therefore the check against this setting is unnecessary?

Reply:
yeah, scroll to the center instead of scrolling that will pin it to the bottom